DevoxxClientMobile/src/main/java/com/devoxx/views/cell/ConferenceCell.java

```diff
@@ -135,15 +143,14 @@ public void updateItem(Conference item, boolean empty) {
             imageTask = new ETagImageTask("conference_" + item.getId(), item.getImageURL());
             imageTask.setOnSucceeded(e -> {
                 background.setImage(imageTask.getValue());
+                fitImage();
             });
             imageTask.exceptionProperty().addListener((o, ov, nv) -> {
                 LOG.log(Level.SEVERE, nv.getMessage());
             });
             executor.submit(imageTask);
             imageTask.image().ifPresent(background::setImage);
```

Comment:
Does `fitImage()` need to be called each time the background image is set?

Reply:
Yes, as each image has different dimensions

Reply:
What I meant is that a call to fitImage is missing then at this line, inside the ifPresent lambda.

Reply:
From what I've tested, the image is always set on succeed of the task (`imageTask.image()` is always empty).